packages/fxa-shared/lib/sentry.ts

```diff
@@ -127,22 +130,36 @@ SentryMetrics.prototype = {
   /**
    * Configure the SentryMetrics instance for this singleton.
    *
-   * @param {String} dsn
-   * @param {String} [release] - settings release version
+   * @param {SentryConfigOpts} config
    */
-  configure(dsn: string, release: string) {
+  configure(config: SentryConfigOpts) {
+    const release = config.release;
     this._logger.info('release: ' + release);
     this._release = release;
-    if (!dsn) {
+    if (!config.sentry?.dsn) {
       this._logger.error('No Sentry dsn provided');
       return;
     }
 
     try {
+      const opts = buildSentryConfig(config, this._logger);
       Sentry.init({
-        release,
-        dsn,
-        beforeSend,
+        ...opts,
+        beforeSend(event: Sentry.Event) {
+          event = beforeSend(event);
+          if (opts.clientName) {
+            event = tagFxaName(event, opts.clientName);
+          } else {
+            event = tagFxaName(event, opts.serverName);
+          }
+          return event;
+        },
+        integrations: [
+          // @ts-ignore
```

Comment:
Why do we need `// @ts-ignore` here?

Reply:
Bug in sentry TS. The lib didn't think browser tracing is a valid integration even though it is.